main.go

```diff
@@ -351,9 +351,12 @@ func main() {
 		log.Fatalf("index.html not found in webapp directory %s", webAppDir)
 	}
 	router.AddRawHTTPHandler("/app*", http.MethodGet, handler.staticHandler)
-	router.AddRawHTTPHandler("/favicon.ico", http.MethodGet, func(w http.ResponseWriter, r *http.Request) {
-		http.ServeFile(w, r, path.Join(webAppDir, "favicon.ico"))
-	})
+	files := []string{"favicon.ico", "robots.txt"}
```

Comment:
`files` looks to general, what about `staticFiles` or `socialStaticFiles`

Reply:
sure, done